.coderabbit.yml

```
@@ -0,0 +1,120 @@
+# CodeRabbit configuration for comprehensive repository review
+reviews:
+  auto_review: true
+  
+  # Request review of all files, not just changed ones
+  review_all: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)